lib/node_modules/@stdlib/blas/ext/base/sapx/README.md

```diff
@@ -50,34 +50,30 @@ The function has the following parameters:
 -   **x**: input [`Float32Array`][@stdlib/array/float32].
 -   **stride**: index increment.
 
-The `N` and `stride` parameters determine which elements in `x` are accessed at runtime. For example, to add a constant to every other element
+The `N` and `stride` parameters determine which elements in the strided array are accessed at runtime. For example, to add a constant to every other element
 
 ```javascript
 var Float32Array = require( '@stdlib/array/float32' );
-var floor = require( '@stdlib/math/base/special/floor' );
 
 var x = new Float32Array( [ -2.0, 1.0, 3.0, -5.0, 4.0, 0.0, -1.0, -3.0 ] );
-var N = floor( x.length / 2 );
 
-sapx( N, 5.0, x, 2 );
+sapx( 4, 5.0, x, 2 );
 // x => <Float32Array>[ 3.0, 1.0, 8.0, -5.0, 9.0, 0.0, 4.0, -3.0 ]
 ```
 
 Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
 
 ```javascript
 var Float32Array = require( '@stdlib/array/float32' );
-var floor = require( '@stdlib/math/base/special/floor' );
 
 // Initial array...
 var x0 = new Float32Array( [ 1.0, -2.0, 3.0, -4.0, 5.0, -6.0 ] );
 
 // Create an offset view...
 var x1 = new Float32Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
-var N = floor( x0.length/2 );
 
 // Add a constant to every other element...
-sapx( N, 5.0, x1, 2 );
+sapx( 3, 5.0, x1, 2 );
 // x0 => <Float32Array>[ 1.0, 3.0, 3.0, 1.0, 5.0, -1.0 ]
 ```
 
```

Comment:
Below: "...in a single-precision floating-point strided array using alternative indexing semantics."